src/hardware/input/mouse_config.cpp

```diff
@@ -127,56 +116,132 @@ bool MouseConfig::ParseCOMModel(const std::string_view model_str,
 		auto_msm = false;
-		return true;
 	} else if (model_str == model_com_3button_str) {
 		model    = MouseModelCOM::Logitech;
 		auto_msm = false;
-		return true;
 	} else if (model_str == model_com_wheel_str) {
 		model    = MouseModelCOM::Wheel;
 		auto_msm = false;
-		return true;
 	} else if (model_str == model_com_msm_str) {
 		model    = MouseModelCOM::MouseSystems;
 		auto_msm = false;
-		return true;
 	} else if (model_str == model_com_2button_msm_str) {
 		model    = MouseModelCOM::Microsoft;
 		auto_msm = true;
-		return true;
 	} else if (model_str == model_com_3button_msm_str) {
 		model    = MouseModelCOM::Logitech;
 		auto_msm = true;
-		return true;
 	} else if (model_str == model_com_wheel_msm_str) {
 		model    = MouseModelCOM::Wheel;
 		auto_msm = true;
-		return true;
+	} else {
+		return false;
 	}
 
-	return false;
+	return true;
+}
+
+static void SetCaptureType(const std::string_view capture_str)
+{
+	if (capture_str == capture_type_seamless_str) {
+		mouse_config.capture = MouseCapture::Seamless;
+	} else if (capture_str == capture_type_onclick_str) {
+		mouse_config.capture = MouseCapture::OnClick;
+	} else if (capture_str == capture_type_onstart_str) {
+		mouse_config.capture = MouseCapture::OnStart;
+	} else if (capture_str == capture_type_nomouse_str) {
+		mouse_config.capture = MouseCapture::NoMouse;
+	} else {
+		assert(false);
+	}
 }
 
-bool MouseConfig::ParsePS2Model(const std::string_view model_str, MouseModelPS2& model)
+static void SetPS2Model(const std::string_view model_str)
 {
 	if (model_str == model_ps2_standard_str) {
-		model = MouseModelPS2::Standard;
+		mouse_config.model_ps2 = MouseModelPS2::Standard;
 	} else if (model_str == model_ps2_intellimouse_str) {
-		model = MouseModelPS2::IntelliMouse;
+		mouse_config.model_ps2 = MouseModelPS2::IntelliMouse;
 	} else if (model_str == model_ps2_explorer_str) {
-		model = MouseModelPS2::Explorer;
-	} else if (auto as_bool = parse_bool_setting(model_str);
-	           as_bool && *as_bool == false) {
-		model = MouseModelPS2::NoMouse;
+		mouse_config.model_ps2 = MouseModelPS2::Explorer;
+	} else if (model_str == model_ps2_nomouse_str) {
+		mouse_config.model_ps2 = MouseModelPS2::NoMouse;
 	} else {
-		return false;
+		assert(false);
 	}
-	return true;
 }
 
-const std::vector<uint16_t> &MouseConfig::GetValidMinRateList()
+static void SetCOMModel(const std::string_view model_str)
 {
-	return list_rates;
+	[[maybe_unused]] const auto result = MouseConfig::ParseCOMModel(
+	        model_str, mouse_config.model_com, mouse_config.model_com_auto_msm);
+	assert(result);
+}
+
+static void SetSensitivity(const std::string_view sensitivity_str)
+{
+	// Coefficient to convert percentage in integer to float
+	constexpr float coeff = 0.01f;
+
+	// Split input string into values
+	auto values_str = split(sensitivity_str, " \t,;");
+	if (values_str.size() > 2) {
+		LOG_WARNING("CONFIG: Too many values in 'mouse_sensitivity', using defaults");
```

Comment:
`using '100,100'` for consistency

Reply:
OK, changed - but the default in log is taken from the constant and actually applied.

Reply:
That's fine, I'm approaching these messages from the perspective of the regular user who will only have access to the config descriptions and the user manual. So if the config description say "100,100 is the default", I think the most useful and least confusing thing to say in the logs is "using 100,100" (instead of "using defaults" or "using X" where X is the actual value). The idea is thst we inform the user that an invalid setting was provided, therefore we operate now as if the default "100,100" setting wasprovided manually. I find that very clear.